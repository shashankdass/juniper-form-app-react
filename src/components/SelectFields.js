import React, { Component } from 'react';
import { Col, FormGroup, Label, Input } from 'reactstrap';

class SelectFields extends Component{
  constructor(props) {
    super(props);
    this.state = {value: ''};
  }

  render() {
    return(
      <div>
      <FormGroup row>
         <Label for="SelectFields" sm={2}>Select Fields:</Label>
         <Col sm={5}>
           <Input type="select"  name="selectOption" id="selectOption">
             <option value="empty"></option>
             <option value="time">Time</option>
             <option value="source_vn">Source</option>
             <option value="destination_vn">Destination</option>
             <option value="port">Port</option>
           </Input>
          </Col>
       </FormGroup>
      </div>
    )
  }
}

export default SelectFields;
