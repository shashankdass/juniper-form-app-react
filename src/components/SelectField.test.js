import React from 'react';
import SelectFields from './SelectFields';
import renderer from 'react-test-renderer';

test('it renders EndTime Component', () => {
  const component = renderer.create(
    <SelectFields></SelectFields>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
