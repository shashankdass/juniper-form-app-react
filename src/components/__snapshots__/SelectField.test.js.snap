exports[`test it renders EndTime Component 1`] = `
<div>
  <div
    className="row form-group">
    <label
      className="col-sm-2 col-form-label"
      htmlFor="SelectFields">
      Select Fields:
    </label>
    <div
      className="col col-sm-4">
      <select
        className="form-control"
        id="selectOption"
        name="selectOption">
        <option
          value="empty" />
        <option
          value="time">
          Time
        </option>
        <option
          value="source_vn">
          Source
        </option>
        <option
          value="destination_vn">
          Destination
        </option>
        <option
          value="port">
          Port
        </option>
      </select>
    </div>
  </div>
</div>
`;
